### PR TITLE
Updates TLS/SSL policy with docs/desc/remediation

### PR DESCRIPTION
**Pull request by scottford-io**
Some new docs/desc/remediations for the TLS/SSL policy

### CODE REVIEW

File: core/mondoo-tls-security.mql.yaml [hunk 1]
```diff
@@ -93,70 +110,166 @@ queries:
         default: score(0);
       }
   - uid: mondoo-tls-security-cert-no-certs-expired
-    title: None of the certificates (intermediate, root) have expired
+    title: Ensure none of the certificates (intermediate, root) have expired
```
Comment (misterpantz) [inline core/mondoo-tls-security.mql.yaml:113]:
```suggestion
    title: None of the certificates (intermediate or root) have expired
```
No other titles start with "Ensure" -- which is a good thing! Eventually no policies will have that.

File: core/mondoo-tls-security.mql.yaml [hunk 2]
```diff
@@ -93,70 +110,166 @@ queries:
...
+      remediation: |
+        Remove RC4 from the server’s list of supported ciphers. Configure preferred ciphers to use modern, secure options such as AES-GCM.
     mql: |
       tls.ciphers.none( /rc4/i )
   - uid: mondoo-tls-security-no-null-cipher-suites
     title: Avoid NULL cipher suites
+    impact: 100
+    docs:
+      desc: |
+        NULL cipher suites do not encrypt data during transmission, leaving it exposed to interception and unauthorized access. Their use undermines the purpose of secure communication.
```
Comment (misterpantz) [inline core/mondoo-tls-security.mql.yaml:186]:
```suggestion
        NULL cipher suites do not encrypt data during transmission, exposing the data to interception and unauthorized access. Their use undermines the purpose of secure communication.
```

File: core/mondoo-tls-security.mql.yaml [hunk 3]
```diff
@@ -93,70 +110,166 @@ queries:
...
     mql: |
       tls.ciphers.none( /null/i )
   - uid: mondoo-tls-security-no-export-cipher-suites
     title: Avoid export ciphers suites
+    impact: 95
+    docs: 
+      desc: |
+        Export cipher suites are intentionally weakened cryptographic algorithms designed to meet outdated export regulations. They are vulnerable to brute-force attacks and compromise encryption security.
+      remediation: |
+        Remove export cipher suites from your server configuration. Replace them with strong modern cipher suites such as AES-GCM or ChaCha20.
```
Comment (misterpantz) [inline core/mondoo-tls-security.mql.yaml:198]:
```suggestion
        Remove export cipher suites from your server configuration. Replace them with strong, modern cipher suites such as AES-GCM or ChaCha20.
```

**Review by misterpantz — COMMENTED** [review]
Reviewed at @tas50's request. Looks excellent. Just a couple tiny things.

**Review by tas50 — APPROVED** [review]

**Comment by charlesjohnson** [comment]
This resolves https://mondoo.productboard.com/entity-detail/features/30132881

**Pull request merged**